scripts/start-on-boot.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -ex
+here=$(dirname $(dirname $(realpath $0)))
+target=/etc/systemd/system/montagu-ci.service
+read -p "What user should the service run as? " user
+
+set -ex
+cp $here/scripts/montagu-ci.service $target
+sed -i "s:__PATH__:$here:g" $target
+sed -i "s:__USER__:$user:g" $target
+systemctl enable montagu-ci
+systemctl start montagu-ci
```

Comment:
I did not expect this to start the system as well as set it up.  Can we just echo start instructions rather than starting teamcity?

Reply:
https://github.com/vimc/montagu-ci/pull/13/commits/9b7673f6384c2de53261ab5f56625662e9440ada